src/main/java/org/dsa/iot/alarm/AlarmWatch.java

```diff
@@ -86,6 +88,13 @@ protected void execute() {
     }
 
     /**
+     * How long in millis since the change of state was first detected.
+     */
+    long getAlarmDetectedDuration() {
```

Comment:
Would it be more clear if the name contained something like `elapsedTimeSinceAlarmTriggered` or something similar? Duration doesn't really imply that it's comparing with "now"